engine/include/cubos/engine/gizmos/plugin.hpp

```diff
@@ -19,21 +19,25 @@ namespace cubos::engine
     /// ## Resources
     /// - @ref Gizmos - used to queue gizmo draw commands.
     ///
-    /// ## Startup tags
-    /// - `cubos.gizmos.init` - the gizmos renderer is initialized, after `cubos.window.init`
-    ///
-    /// ## Tags
-    /// - `cubos.gizmos.input` - gizmos interaction is handled, after `cubos.window.poll` and
-    ///   before `cubos.gizmos.draw`.
-    /// - `cubos.gizmos.draw` - queued gizmos are rendered to the window, after `cubos.renderer.draw` and
-    ///   before `cubos.window.render`.
-    /// - `cubos.gizmos.pick` - the ScreenPicker resource is accessed to detect gizmos at mouse coordinates, after
-    /// `cubos.gizmos.draw`
-    ///
     /// ## Dependencies
     /// - @ref screen-picker-plugin
     /// - @ref window-plugin
 
+    /// @brief the gizmos renderer is initialized, after `cubos.window.init`
```

Comment:
```suggestion
    /// @brief Gizmos renderer is initialized, after @ref NewTagNameHere.
```
Please follow this style for all other tags.